core/templates/core/basic_search_page.html

```diff
@@ -129,10 +144,23 @@ <h3 class="sr-only">Layout</h3>
 
         let return_quick_init_vname = "{{ return_quick_init_vname }}";
         {%  if return_quick_init_vname %}
-        let vname_action_url = "{% url return_quick_init_vname '__entry_id__' %}";
+            let vname_action_url = "{% url return_quick_init_vname '__entry_id__' %}";
         {% endif %}
         let recref_mode = "{{ recref_mode }}";
 
+        $('#export_excel_btn').click(function (e){
+            let msg = 'Are you sure export Excel?'
+            let num_records = $('#num_records').val()
+            if (num_records) {
+                msg = 'Are you sure export ' + num_records + ' records in Excel?'
+            }
```

Comment:
'Are you sure export Excel?' => 'Are you sure you want to export the search results to Excel?'

'Are you sure export ' + num_records + ' records in Excel? '=> 'Are you sure you want to export {num_records} search results to Excel?'